certbot-dns-cloudflare/setup.py

```diff
@@ -7,7 +7,9 @@
 version = '2.11.0.dev0'
 
 install_requires = [
-    'cloudflare>=1.5.1',
+    # for now, do not upgrade to cloudflare>=2.20 to avoid deprecation warnings and the breaking
```

Comment:
Is there an expected time frame where we can upgrade this or is this considered indefinite for now?

Reply:
i think it's indefinite for now, but that also might change pretty soon if that makes sense? according to [their announcement](https://github.com/cloudflare/python-cloudflare/discussions/191#discussion-6462999) `cloudflare` 3.0 was supposed to be released last friday but it's still not out yet

as of today, the latest version of the cloudflare package prints warning messages to stderr on every run and their advice is to pin back the dependency

there's more links and discussion about this in the github issue https://github.com/certbot/certbot/issues/9938

Reply:
Okay, we can just keep watch out for future releases